pkg/agent/config/config.go

```diff
@@ -319,6 +321,9 @@ func isValidResolvConf(resolvConfFile string) bool {
 	if err := scanner.Err(); err != nil {
 		return false
 	}
+	if !foundNameserver {
+		return false
+	}
 	return true
```

Comment:
nit: just return the bool
```suggestion
	return foundNameserver
```